spalp_example/spalp_example.module

```diff
@@ -5,7 +5,7 @@
  * Contains spalp_example.module.
  */
 
-use Drupal\Core\Routing\RouteMatchInterface;
+define("SPALP_EXAMPLE_APP_ID", "spalp_example");
```

Comment:
please use single quotes unless we need double quotes: https://www.drupal.org/docs/develop/standards/coding-standards#quotes